### PR TITLE
Add bitlen and app_params_get opcode

### PR DESCRIPTION
**Pull request by vuvth**
### Changed
- Add bitlen opcode and unit test. 
- Add app_params_get and unit test

### CODE REVIEW

File: packages/runtime/test/src/interpreter/opcode-list.ts [hunk 1]
```diff
@@ -5875,4 +5875,38 @@ describe("Teal Opcodes", function () {
...
+    });
+
+    it("should work with number", () => {
+      stack.push(8n);
+      const op = new BitLen([], 1, interpreter);
+      op.execute(stack);
+      assert.equal(stack.pop(), 4n);
+    });
+
+    it("shoud work with a shore byte array", () => {
```
Comment (ratik21) [inline packages/runtime/test/src/interpreter/opcode-list.ts:5894]:
```suggestion
    it("shoud work with a short byte array", () => {
```

File: packages/runtime/test/src/interpreter/opcode-list.ts [hunk 2]
```diff
@@ -5875,4 +5875,38 @@ describe("Teal Opcodes", function () {
...
+      const byte = "abcd";
+      const op = new BitLen([], 1, interpreter);
+
+      stack.push(parsing.stringToBytes(byte));
+      op.execute(stack);
+      assert.equal(stack.pop(), 31n);
+    });
+
+    it("shoud work with a long byte array", () => {
+      const byte = "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
```
Comment (ratik21) [inline packages/runtime/test/src/interpreter/opcode-list.ts:5904]: you can do `const byte = "f".repeat(n)`

File: packages/runtime/test/src/interpreter/opcode-list.ts [hunk 3]
```diff
@@ -5875,4 +5875,38 @@ describe("Teal Opcodes", function () {
...
+      const op = new BitLen([], 1, interpreter);
+
+      stack.push(parsing.stringToBytes(byte));
+      op.execute(stack);
+      assert.equal(stack.pop(), 623n);
+    });
```
Comment (ratik21) [inline packages/runtime/test/src/interpreter/opcode-list.ts:5910]: Where are these tests referenced from? We need to add few corner cases as well. You can reference `go-algorand` tests.

**Review by ratik21 — COMMENTED** [review]
Changelog needs to be updated

File: packages/runtime/test/src/interpreter/opcode-list.ts (hunk 1 above)
Comment (surbhit14) [reply packages/runtime/test/src/interpreter/opcode-list.ts:5894]: isn't "shoud work with any  short byte array" better

File: CHANGELOG.md [hunk 4]
```diff
@@ -13,6 +13,7 @@
 ### Bug Fixes
 
 - Int Pseudo-Ops can't start with 0x(hex) or 0(oct) prefix. (#562)
+- Add missiing opcode `bitlen` and `app_params_get`.
```
Comment (ratik21) [inline CHANGELOG.md:16]:
```suggestion
- Add missing opcodes `bitlen` and `app_params_get`.
```

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 5]
```diff
@@ -4234,3 +4235,120 @@ export class Log extends Op {
...
+   * @param interpreter interpreter object
+   */
+  constructor (args: string[], line: number, interpreter: Interpreter) {
+    super();
+    this.line = line;
+    this.interpreter = interpreter;
+    assertLen(args.length, 1, line);
+
+    if (!AppParamDefined[interpreter.tealVersion].has(args[0])) {
+      throw new RuntimeError(RUNTIME_ERRORS.TEAL.UNKNOWN_ASSET_FIELD, {
```
Comment (ratik21) [inline packages/runtime/src/interpreter/opcode-list.ts:4298]: `UNKNOWN_ASSET_FIELD` ?

File: packages/runtime/src/parser/parser.ts [hunk 6]
```diff
@@ -254,7 +255,7 @@ const interpreterReqList = new Set([
   "app_local_put", "app_global_put", "app_local_del", "app_global_del",
   "gtxns", "gtxnsa", "min_balance", "gload", "gloads", "callsub", "retsub",
   "gaid", "gaids", "loads", "stores", "itxn_begin", "itxn_field", "itxn_submit",
-  "itxn", "itxna", "txnas", "gtxnas", "gtxnsas", "args", "log"
+  "itxn", "itxna", "txnas", "gtxnas", "gtxnsas", "args", "log", 'bitlen', 'app_params_get'
```
Comment (ratik21) [inline packages/runtime/src/parser/parser.ts:258]: I think `bitlen` does not need to be in "interpreter req list"

File: packages/runtime/src/interpreter/opcode-list.ts (hunk 5 above)
Comment (vuvth) [reply packages/runtime/src/interpreter/opcode-list.ts:4298]: Thanks! My bad.

**Review by ratik21 — APPROVED** [review]
